scripts/utils/import-metrics-mimirtool.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# set kubeconfig
+export KUBECONFIG=$1
+
+# run mimirtool, keep shell open
+screen -t -s -d -m bash -c "kubectl -n cattle-monitoring-system run mimirtool --rm -ti --image=grafana/mimirtool:2.13.0 --env config.expand-env=true --env querier.max-samples=100000000 --command -- sh" &
```

Comment:
Do we actually need this to use `screen`? Couldn't we deploy mimirtool as a pod?